### ISSUE TITLE
Fix e2e tests

### ISSUE THREAD

**Issue by antonmedv** [deployphp/deployer#2671]
Hi, @bsacharski 
I broke e2e tests, please, take a look.
I only added: 
```
usermod -a -G www-data deployer
```

**Comment by bsacharski** [comment]
@antonmedv no problem I will have a look, though it'll have to wait till Monday. I have no access to any pc at the moment.

**Comment by bsacharski** [comment]
@antonmedv 
I've checked the cause for the failure. When the `deployer` container tries to fetch `http://server.test` we get response 500 with following body:
```
/project # curl 'http://server.test'
<!-- The stream or file &quot;/var/www/html/releases/2/storage/logs/laravel.log&quot; could not be opened in append mode: failed to open stream: Permission denied (500 Internal Server Error) -->
...
```

I've checked the permissions inside the `server` container and got following:
```
root@d530c83d1125:/var/www/html/releases/2/storage# ls -la
total 20
drwxr-xr-x 5 deployer www-data 4096 Sep 27 17:34 .
drwxr-xr-x 3 deployer deployer 4096 Sep 27 17:34 ..
drwxr-xr-x 3 deployer www-data 4096 Sep 27 17:34 app
drwxr-xr-x 6 deployer www-data 4096 Sep 27 17:34 framework
drwxr-xr-x 2 deployer www-data 4096 Sep 27 17:34 logs
```

So the `storage` directory (alongside its subdirectories/files) is set ot be writable only by owner (the `deployer` user).
PHP works with the`www-data` user, so it cannot write to log file, which causes the failure.

tl;dr; user `www-data` needs to be able to write to `storage` directory and its subdirectories.

**Comment by antonmedv** [comment]
Thanks a lot!
